Deploy/stacks/AI4PublicHealth/Stack_Deployment/README.md

```diff
@@ -0,0 +1,170 @@
+
+## 3.2) Greenspace (.shp) and Points of Interest (.csv)
+The [OS Open Greenspace] dataset provides a comprehensive mapping of greenspaces across Great Britain, including public parks, sports facilities, and allotments, along with access points. Similarly, the [Points of Interest (POI)] dataset includes detailed information about commercial and non-commercial establishments, such as businesses, retail spaces, and service providers, with attributes like phone numbers, web URLs, and classifications. These datasets are crucial for spatial analyses related to urban planning, service accessibility, and community health.
+
+### 1) Stack data uploader
+Similar to the Food Hygiene Rating data, the instantiation of these datasets also relies on the [Stack Data Uploader]. Copy configuration files from `config` to `Deploy/stacks/dynamic/stack-data-uploader/inputs/config/`, and replace the `readme.txt` files in the `data` sub-folders with the greenspace shapefiles and POI CSV files. Also, copy the files (.obda) from [mapping folder] and paste to `Deploy/stacks/dynamic/stack-data-uploader`.
+
+Finally, navigate to `Deploy/stacks/dynamic/stack-data-uploader` and execute:
+
+```bash
+bash ./stack.sh start Healthcare
+```
+### 2) SPARQL query via virtual knowledge graph
+
+The Ontop endpoint is also employed to support querying Greenspace and Points of Interest data semantically. Using [OntoGreenspace] and [OntoPOI] along with defined OBDA mappings, the endpoint semantically represents relational database records, including greenspace locations, access points, business details, and geographic coordinates.
+
+## 3.3) GPS Trajectories (.csv)
+Individual GPS trajectories used in this study are instantiated using the [Fenland Trajectory Agent]. This agent extracts key attributes such as latitude, longitude, speed, heading, and time from CSV files, organises them into triples using the Ontology of Devices ([OntoDevice]), and uploads the data into the knowledge graph and a relational database. Details on deploying this agent can be found [here].
+
+# 4. Visualisation
+The visualisation for this project is powered by the TWA Visualisation Platform ([TWA-VP]). To deploy the TWA-VP stack, first copy the folder named `config` from the `public` folder located at `Deploy/stacks/AI4PublicHealth/TWA-VP/public/` and paste it into the directory `web/twa-vis-platform/code/public/`. Next, copy the folder named `Fenland` from `Deploy/stacks/AI4PublicHealth/TWA-VP/public/images/` and paste it into `web/twa-vis-platform/code/public/images/`. Then, navigate to the `web/twa-vis-platform` directory and create the following files:
+
+- `mapbox_username`
+- `mapbox_api_key`
+
+Populate these files with your Mapbox username and API key. After that, execute the following command in the terminal:
+
+```bash
+docker compose up -d
+```
+The default port for the TWA-VP is 3000. Please ensure this port is open before deploying the visualisation stack.
+
```

Comment:
I think it is important to add here one critical piece of information: Currently, the visualisation is configured with hard-coded IP addresses as stack data sources. This implies that, even if one follows all the data instantiation and uploading steps in section 3 above, by default, this visualisation will not display that, but instead it will display what is being served by the hard-coded IP addresses (if these are up and running!). If one wants to change that, one needs to edit the visualisation configuration files. Please make this very clear.

Reply:
Thanks for this constructive suggestions. I have added necessary descriptions and provided guidance about where need to be changed if user want to display the own data. Please check commit [8d669e3f9bdea5c46c3e3e226ef0af19ff8c9aab]